docs/guides/joint_fabric_guide.md

```diff
@@ -0,0 +1,253 @@
+# Joint Fabric Guide
+
+-   [Joint Fabric Guide](#joint-fabric-guide)
+    -   [Joint Fabric Example Applications](#joint-fabric-example-applications)
+        -   [Bootstrap Joint Fabric Demo on Linux](#bootstrap-joint-fabric-demo-on-linux)
+        -   [Run Joint Fabric Demo](#run-joint-fabric-demo)
+
+## Joint Fabric Example Applications
+
+jf-control-app and jf-admin-app example applications are provided to demonstrate
+the Joint Fabric feature. You can find them in the examples folder.
+
+![matter_joint_fabric](images/matter_joint_fabric.png)
+
+jf-control-app acts as a commissioner and control plane for jf-admin-app. It
```

Comment:
Might call out that jf-control-app is acting as the JF Anchor and so it too has the JF Administrator Cluster and the JF Datastore Cluster, right?

Reply:
Nope: jf-admin-app acts as JF Anchor so it has an instance of the JF Administrator Cluster and of the JF Datastore Cluster. jf-control-app has an operational identify and a NOC with Administrator CAT but those are used only for the initial onboarding of the jf-admin-app. Details, links to relevant documents, architecture design can be found in the PR description, pls check.